cmake/configs/default.cmake

```diff
@@ -232,7 +232,12 @@ hunter_default_version(glbinding VERSION 2.1.3-p0)
 hunter_default_version(glew VERSION 2.0.0-p1)
 hunter_default_version(glfw VERSION 3.3.0-p4)
 hunter_default_version(glib VERSION 2.54.0)
-hunter_default_version(glm VERSION 0.9.9.0)
+if(ANDROID)
+    #bug in the latest glm on Android
```

Comment:
Please use indentation 2 as in the other parts of this file, e.g.
* https://github.com/ruslo/hunter/blob/v0.23.35/cmake/configs/default.cmake#L66-L72

Reply:
its changed